protected/views/patient/crud/_form.php

```diff
@@ -586,6 +586,7 @@ function getScenarioFromCode() {
     return {'1': 'referral', '2': 'self_register', '0': 'other_register'};
   }
 
+  var dup_checked = false;
```

Comment:
This variable only ever seems to be set to "true". Shouldn't it be set to false if the patient name or date of birth is changed after they've acknowledged the conflict?

Reply:
Yep, in line 610, it was set to be true after the "Acknowledge" button clicked.